src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java

```diff
@@ -105,7 +111,8 @@ protected String getWidthResolution() {
 
     /**
      * Gets selected Height resolution for video converter.
-     * @return int, Height selected of Resolution.
+     *
+     * @return an int, Height selected of Resolution.
```

Comment:
It would be better to put: "an int with selected resolution height"

Reply:
ok ray, i correct it